canonical-web-and-design/ubuntu.com
# Load static data for vendor pages

## Done

I took a guess looking at the [design](https://app.zeplin.io/project/60c1ca887b99f18a380e57e9/screen/60f169f4a5400b0f1506c19f), but obviously if the copy content they give us does not fit this data structure, we'll have to make changes.

But I thought this would be the best way to unblock this issue.

If @bethcollins92 thinks we can work with this, let's go ahead and review and merge, if not feel free to close this PR or suggest changes.

## QA

Best way to QA this at this point is to do it locally, as we don't have any content.

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
- Add `{{vendors_strip}}` to the `search-results.html` template, preferrably in the header, so you can see the json data clearly.
- Do the same for any category views e.g. `desktops.html` (the variable should be available in any of these templates)


## Issue / Card

Fixes https://github.com/canonical-web-and-design/web-squad/issues/4260

## Screenshots
